airflow/www/decorators.py

```diff
@@ -125,3 +127,29 @@ def wrapper(self, *args, **kwargs):
                                         __class__.__name__ + ".login"))
         return wrapper
     return decorator
+
+
+def log_webserver_stats(endpoint, log_views=False, log_duration=False, log_failures=False):
+    """
+    Decorator to log server side stats from webserver UI.
+
+    Currently supports logging views and load failures as counters
+    and load duration as timing.
+    """
+    def decorator(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            start_dttm = timezone.utcnow()
+            if (log_views):
```

Comment:
```suggestion
            if log_views:
```
These brackets are unnecessary

Reply:
Done.